src/engraving/dom/slurtie.cpp

```diff
@@ -164,6 +159,15 @@ void SlurTieSegment::spatiumChanged(double oldValue, double newValue)
     }
 }
 
+void SlurTieSegment::cloneLdata(SlurTieSegment* seg) const
+{
+    for (int i = 0; i < int(Grip::GRIPS); ++i) {
+        seg->m_ups[i]   = m_ups[i];
+        seg->m_ups[i].p = PointF();
+    }
+    seg->mutldata()->path.set_value(ldata()->path());
```

Comment:
I understand the issue and your solution, but why do we need to copy this data at all? The path of a SlurTieSegment is recalculated at every layout, which means any information that gets copied here will be overwritten immediately after. If we want to copy the curve, we should just copy the GRIP points (as we already do) and the path will be reconstructed from those. The path is basically a set of instructions for the Painter to draw the curve, so I really don't think it should be copied

Reply:
Argh you're totally right, this line is completely useless!